backend/src/main/java/gov/cdc/usds/simplereport/api/patient/PatientResolver.java

```diff
@@ -26,30 +27,40 @@ public PatientResolver(PersonService ps, OrganizationService os) {
   }
 
   // authorization happens in calls to PersonService
+  // will update as part of #6062
   @QueryMapping
   public List<Person> patients(
       @Argument UUID facilityId,
       @Argument int pageNumber,
       @Argument int pageSize,
-      @Argument boolean includeArchived,
+      @Argument Boolean includeArchived,
+      @Argument ArchivedStatus archivedStatus,
       @Argument String namePrefixMatch,
       @Argument boolean includeArchivedFacilities) {
+    ArchivedStatus status;
```

Comment:
⚠️ for backward compatibility -- will remove in separate PR once this one is deployed